sdk/react/panels/AudioAndCCSelectionPanel.js

```diff
@@ -0,0 +1,196 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ */
+'use strict';
+
+import React, { Component } from 'react';
+import {
+  Animated,
+  Text,
+  TouchableHighlight,
+  View,
+} from 'react-native';
+
+var animationDuration = 1000;
+var Constants = require('../constants');
+var {
+  BUTTON_NAMES
+} = Constants;
+
+var Utils = require('../utils');
+var styles = require('../utils').getStyles(require('./style/AudioAndCCSelectionPanel'));
+var ItemSelectionScrollView = require('./ItemSelectionScrollView');
+
+var AudioAndCCSelectionPanel = React.createClass({
+  propTypes: {
+    audioTracksTitles: React.PropTypes.array,
+    selectedAudioTrackTitle: React.PropTypes.string,
+    closedCaptionsLanguages: React.PropTypes.array,
+    selectedClosedCaptionsLanguage: React.PropTypes.string,
+    onSelectAudioTrack: React.PropTypes.func,
+    onSelectClosedCaptions: React.PropTypes.func,
+    onDismiss: React.PropTypes.func,
+    width: React.PropTypes.number,
+    height: React.PropTypes.number,
+    config: React.PropTypes.object
+  },
+
+  getInitialState: function() {
+    return {
+      opacity: new Animated.Value(0)
+    };
+  },
+
+  componentDidMount:function () {
+    this.state.opacity.setValue(0);
+    Animated.parallel([
+      Animated.timing(
+        this.state.opacity,
+        {
+          toValue: 1,
+          duration: animationDuration,
+          delay: 0
+        }),
+    ]).start();
+  },
+
+  onAudioTrackSelected: function(name) {
+    if (this.props.selectedAudioTrackTitle !== name) {
+      this.props.onSelectAudioTrack(name);
+    }
+  },
+
+  onClosedCaptionsLanguageSelected: function(name) {
+    var offButtonLocalizedTitle = Utils.localizedString(this.props.config.locale, "Off", this.props.config.localizableStrings);
+
+    if (name == offButtonLocalizedTitle) {
+      this.props.onSelectClosedCaptions("");
+    } else if (this.props.selectedClosedCaptionsLanguage !== name) {
+      this.props.onSelectClosedCaptions(name);
+    }
+  },
+
+  onDismissPress: function() {
+    this.props.onDismiss();
+  },
+
+  renderHeaderView: function(hasMultiAudioTracks, hasClosedCaptions) {
+    var leftTitle;
+    var rightTitle;
+
+    var localizedAudioTitle = Utils.localizedString(this.props.config.locale, "Audio", this.props.config.localizableStrings);
+    var localizedSubtitlesTitle = Utils.localizedString(this.props.config.locale, "Subtitles", this.props.config.localizableStrings);
```

Comment:
We are using Closed Caption as naming convention. Can we continue doing that instead "Subtitles" ?